docs/workflows/doi.md

```diff
@@ -1,4 +1,120 @@
+| Grid              | `digital_science.gridYYYYMMDD`                      | GRID      |
++-------------------+-----------------------------------------------------+-----------+
+| MAG               | `mag.*YYYYMMDD` (multiple individual tables)        | GRID/DOI  |
++-------------------+-----------------------------------------------------+-----------+
+| Open Citations    | `open_citations.open_citationsYYYYMMDD`             | DOI       |
++-------------------+-----------------------------------------------------+-----------+
+| ORCID             | `orcid.orcid`                                       | ORCID/DOI |
++-------------------+-----------------------------------------------------+-----------+
+| Unpaywall         | `our_research.unpaywall`                            | DOI       |
++-------------------+-----------------------------------------------------+-----------+
+| iso3166           | `ios.iso3166_countries_and_regions`                 | Country   |
++-------------------+-----------------------------------------------------+-----------+
+```
+
+## Synthesis
+
+After fetching the datasets, they are synthesised to produce aggregate time series statistics for each country and institution (entity type) in the dataset. The aggregate timeseries statistics include publication count, open access status, citation count and alt-metrics. 
+
+The synthesis occurs in three steps: 
+- Creating a table of publications.
+- Grouping the publications by entity type and year of publication.
+- Computing aggregated summaries for each group. Each step of the process is explained below with examples.
+
+``` eval_rst
+.. image:: ../static/diagrams/workflow_summary.png
+   :width: 650
+```
+
+The table of publications is created by joining records from the research publication datasets on Digital Object Identifiers (DOIs); unique digital identifiers given to the majority of publications. Figure 2 illustrates how each dataset contributes to the publications table during the joining process, using the example of a single publication. Unique publications are discovered with Crossref Metadata, from which the publication’s DOI, Journal, Publisher, Funder identifiers and citation counts are derived. The publication’s Open Access status is computed using Unpaywall. The authors of the paper and their institutional affiliations are derived with Microsoft Academic Graph. The Research Organisation Registry (ROR) is used to enrich the institutional affiliation records with institution details and GeoNames maps institutions to countries and regions.  
+
+``` eval_rst
+.. image:: ../static/diagrams/pid_workflow.png
+   :width: 600
+```
+
+Once the publications table has been created, the publications are grouped by entity type and publication year. For instance publications are grouped by institution and publication year. The last step involves creating aggregate timeseries statistics based on the yearly groups of publications.
+
```

Comment:
Should the label 'papers' in this figure be 'publications'?

Reply:
yeah, that is probably correct. The diagram was reused from someone Jamie created for the website. Can update it if that other diagram wants to be changed too?

Reply:
Yup, please do